migrator/src/test/java/com/google/phonenumbers/migrator/MigrationJobTest.java

```diff
@@ -0,0 +1,110 @@
+package com.google.phonenumbers.migrator;
+
+    import static com.google.common.truth.Truth.assertThat;
+
+    import com.google.i18n.phonenumbers.metadata.DigitSequence;
+    import com.google.i18n.phonenumbers.metadata.RangeSpecification;
+    import com.google.i18n.phonenumbers.metadata.RangeTree;
+    import com.google.i18n.phonenumbers.metadata.table.RangeKey;
+    import java.io.FileNotFoundException;
+    import java.io.IOException;
+    import java.nio.file.NoSuchFileException;
+    import java.nio.file.Paths;
+    import java.util.Collections;
+    import org.junit.Assert;
+    import org.junit.Test;
+    import org.junit.runner.RunWith;
+    import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class MigrationJobTest {
+
+  private static final String TEST_DATA_PATH = "./src/test/java/com/google/phonenumbers/migrator/testing/testData/";
+
+  @Test
+  public void testInvalidNumbersFile() {
```

Comment:
For naming tests within Google, we tend to follow the following format which includes the method being tested, (sometimes) specific details of the state and the expected outcome. For example, this test could be named: 

createFromFilepath_invalidPathLocation_expectException